### ISSUE TITLE
How to style Kino.JS and Kino.JS.Live components?

### ISSUE THREAD

**Issue by BrooklinJazz** [livebook-dev/kino#96]
Hi all, forgive me if this isn't the right place to ask this. Please remove the issue if it's not.

How can I effectively style Kino.JS.Live components?

So far I've tried:
- Inline styling ✔
- appending a child to the head. ✔
```js
      const style = document.createElement('style');
      style.type = 'text/css';
      style.innerHTML = '.cssClass { color: red; }';
      document.getElementsByTagName('head')[0].appendChild(style);
```
- await ctx.importCSS("https://cdn.tailwindcss.com"); ❌
- reusing existing class names applied in the livebook ❌

If it's helpful, I would be happy to create an example in the documentation of the recommended method!

**Comment by jonatanklosko** [comment]
Hey @BrooklinJazz! You can define your own CSS files and import them.

[Here](https://github.com/jonatanklosko/kino_lab/blob/main/lib/kino/youtube.ex) is a minimal example with inline assets.

And here's a more elaborate usage with actual files:

https://github.com/livebook-dev/kino/blob/976a9711b83e24d29616b75384378c6ff26f4b5e/lib/assets/data_table/main.js#L1-L8

And the corresponding CSS file [here](https://github.com/livebook-dev/kino/blob/976a9711b83e24d29616b75384378c6ff26f4b5e/lib/assets/data_table/main.css).

Let me know if you run into any issues!

**Comment by jonatanklosko** [comment]
We have the individual pieces in [`Kino.JS`](https://hexdocs.pm/kino/Kino.JS.html) docs, but I agree that an additional note on styling could be useful. We have a section on "Assets", so we can add the note there, with an example that imports a relative main.css file and a random font from google fonts, which should give people a better idea. A PR is definitely welcome! :)

**Comment by jonatanklosko** [comment]
Closed via #97.
